akka-actor/src/main/scala/akka/dispatch/Mailboxes.scala

```diff
@@ -202,8 +206,14 @@ private[akka] class Mailboxes(
         // It doesn't matter if we create a mailbox type configurator that isn't used due to concurrent lookup.
         val newConfigurator = id match {
           // TODO RK remove these two for Akka 2.3
-          case "unbounded" => UnboundedMailbox()
-          case "bounded"   => new BoundedMailbox(settings, config(id))
+          case "unbounded"                          => UnboundedMailbox()
+          case "bounded"                            => new BoundedMailbox(settings, config(id))
+          case _ if id.startsWith("typed-bounded:") =>
```

Comment:
define this prefix as an internal constant, `Mailboxes.TypedBoundedMailboxId`

Reply:
rename to `bounded-capacity:` instead of making it special for typed, nothing we would document though